native-dep-packs/rebuild-precompiled-addons.sh

```diff
@@ -8,9 +8,7 @@ set -eo pipefail
 
 cd `dirname $BASH_SOURCE`
 
-# Maintenance Note:
-# This should be kept in sync with packages/autoprofile/precompile/build-all-addons.js-> ABI_VERSIONS.
-declare -A ABI_VERSIONS=( \
+declare -a ABI_VERSIONS=( \
   ["108"]="18.18.2" \
```

Comment:
Would be really nice if we can add an argument to build a single prebuild version. Must not be part of this PR.